### PR TITLE
Win32 abseil-cpp integration for GNU environment

### PR DESCRIPTION
**Pull request by despair86**
- fix win32 installer nit
- add warning to ryzen user to check their pc for ROM updates
- some limited travis-ci automation for installer

### CODE REVIEW

**Comment by michael-loki** [comment]
abseil changes look sane to me

File: CMakeLists.txt [hunk 1]
```diff
@@ -167,8 +165,8 @@ if(ASAN)
 endif(ASAN)
 
 if(SHADOW)
+  set(WITH_STATIC OFF)
   set(WITH_SHARED ON)
-  add_compile_options(-fPIC)
```
Comment (neuroscr) [inline CMakeLists.txt]: why is win32 messing with Shadow?

Comment (despair86) [reply CMakeLists.txt]: `-[fK][Pp][Ii][Cc]` is set globally due to abseil requiring it

**Comment by majestrate** [comment]
this ready for merge?

**Comment by despair86** [comment]
ya go ahead @majestrate

**Pull request merged**